### PR TITLE
Feature/spm

### PR DESCRIPTION
**Pull request by ThasianX**

### CODE REVIEW

File: Cordate.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -575,7 +482,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```
Comment (powerje) [inline Cordate.xcodeproj/project.pbxproj]: Do we need to change this?

If we do, we need to update the podspec as well

**Review by powerje — CHANGES_REQUESTED** [review]

File: Cordate.xcodeproj/project.pbxproj (hunk 1 above)
Comment (ThasianX) [reply Cordate.xcodeproj/project.pbxproj]: I guess we don't. For our internal dependencies we want iOS 12 and up but I'm not too sure about public ones like this.

Comment (ThasianX) [reply Cordate.xcodeproj/project.pbxproj]: After some thought, makes sense to just lower the deployment target back down as others may have a lower deploy target

File: Cordate/Sources/Core/Cordate.h [hunk 2]
```diff
@@ -1,9 +0,0 @@
-@import UIKit;
```
Comment (powerje) [inline Cordate/Sources/Core/Cordate.h]: Undelete this

Comment (ThasianX) [reply Cordate/Sources/Core/Cordate.h]: done

**Review by powerje — APPROVED** [review]

**Pull request merged**